src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,38 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList';
+
+import './PostInfo.scss';
+
+interface PostInfoProps {
+  post: Post;
+}
+
+export const PostInfo: React.FC<PostInfoProps> = ({ post }) => {
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        {post.user ? <UserInfo user={post.user} /> : null}
+
+      </div>
+
+      <p className="PostInfo__body">
+        {post.body}
+      </p>
+
+      <hr />
+
+      {(post.comments !== null
+        && post.comments !== undefined
+        && post.comments.length > 0)
```

Comment:
```suggestion
      {(post.comments.length > 0)
```
if comments would be null/undefined the value would be falsy, therefore b tag would be rendered, checking length is good enough

Reply:
![possiblyNull](https://github.com/mate-academy/react_static-list-of-posts/assets/137656592/6b53b1a0-6903-4848-85a6-fc80e3c06e48)